.github/workflows/build.yml

```
@@ -0,0 +1,80 @@
+name: Build
+
+on:
+  pull_request_review:
+    types: [submitted, edited]

```

Is `edited` required?

```suggestion
    types: [submitted]
```

---

Probably not.